.travis.yml

```diff
@@ -1,25 +1,69 @@
 sudo: required
 dist: trusty
 language: python
+compiler: gcc
 
 virtualenv:
   system_site_packages: true
 
 python:
   - 3.4
 
+env:
+  global:
+     # - QT_BASE=53
+     # - QT_BASE=54
+     # - QT_BASE=55
+     # - QT_BASE=56
+     - QT_BASE=57
+     #- GCC_ARM=/opt/gcc-arm-none-eabi/bin
+  matrix:
+     # - FLAVOR=ALL
+     - FLAVOR=DEFAULT
+     - FLAVOR=AVR9X
+     # - FLAVOR=9X
+     # - FLAVOR=GRUVIN9X
+     # - FLAVOR=MEGA2560
+     - FLAVOR=ARM9X
+     # - FLAVOR=AR9X
+     # - FLAVOR=SKY9X
+     # - FLAVOR=9XRPRO
+     # - FLAVOR=TARANIS
```

Comment:
To build those a 2nd time?  "TARANIS" includes the X7 vs. "X9" which is just the bigger Tarani.  Is that what you mean?